src/kem/classic_mceliece/pqclean_mceliece348864f_avx/consts.S

```diff
@@ -1,7 +1,10 @@
-.data
+#if defined(__APPLE__)
```

Comment:
Am I blind not finding this in the patch code? Or is it intentionally not part of the patch?

Reply:
That's a change from upstream pqclean.